cognite/client/_api/assets.py

```diff
@@ -1026,8 +1045,10 @@ def list(
             root (bool | None): filtered assets are root assets or not.
             external_id_prefix (str | None): Filter by this (case-sensitive) prefix for the external ID.
             aggregated_properties (Sequence[AggregateAssetProperty] | None): Set of aggregated properties to include. Options are childCount, path, depth.
-            partitions (int | None): Retrieve assets in parallel using this number of workers. Also requires `limit=None` to be passed. To prevent unexpected problems and maximize read throughput, API documentation recommends at most use 10 partitions. When using more than 10 partitions, actual throughout decreases. In future releases of the APIs, CDF may reject requests with more than 10 partitions.
+            partitions (int | None): Retrieve assets in parallel using this number of workers. Also requires `limit=None` to be passed. To prevent unexpected problems and maximize read throughput, API documentation recommends at most use 10 partitions. When using more than 10 partitions, actual throughout decreases. In future releases of the APIs, CDF may reject requests with more than 10 partitions. Note, when using partitions sort is not supported. Since partitions are done independently of sorting, there would be no guarantee of the sort order between elements from different partitions.
```

Comment:
Consider making a separate "note" section regarding this interplay between partitions and sort.

What I mean:
```
Args:
    foo (int): asdf

Returns:
    int: the computed asdf

Note:
    Bla bla
```

Reply:
Might be good, but I think it's getting out of scope for this PR. Actually, since this section is pretty much copy-paste across the different apis. Maybe better to add this to a shared docs section with notes and usage guides?

Reply:
You added functionality that has "special behavior" with certain existing parameters, that seems very much in-scope to me 😄

> Maybe better to add this to a shared docs section with notes and usage guides?

Sounds good 👍